Meadowlark-js/packages/meadowlark-core/src/handler/FrontendFacade.ts

```diff
@@ -23,10 +23,18 @@ import { queryValidation } from '../middleware/ValidateQueryMiddleware';
 import { documentInfoExtraction } from '../middleware/ExtractDocumentInfoMiddleware';
 import { metaEdModelFinding } from '../middleware/FindMetaEdModelMiddleware';
 
-initializeLogging();
-
 type MiddlewareStack = (model: MiddlewareModel) => Promise<MiddlewareModel>;
 
+const moduleName = 'FrontendFacade';
+
+/**
+ * Initialization - anything here should be runnable more than once without impact
+ */
+async function initialize(): Promise<void> {
```

Comment:
Accidentally pulled this small improvement to initialization in. Seemed ok to just include, it defers logging initialization in an attempt to avoid grabbing logging environment variables before dotenv runs.